api/v1beta2/awscluster_types.go

```diff
@@ -221,7 +222,7 @@ type AWSLoadBalancerSpec struct {
 
 	// LoadBalancerType sets the type for a load balancer. The default type is classic.
 	// +kubebuilder:default=classic
-	// +kubebuilder:validation:Enum:=classic;elb;alb;nlb
+	// +kubebuilder:validation:Enum:=classic;elb;alb;nlb;disabled
```

Comment:
One thing that I think would be helpful is having a webhook validation that looks to see if the other related LoadBalancer options are not set. 

For example, I shouldn't be able to set things like `Scheme, HealthCheckProtocol, AdditionalSecurityGroups` and so on if the loadbalancertype == disabled

Reply:
Implemented.